frontend/public/components/machine-set.tsx

```diff
@@ -4,15 +4,32 @@ import { Link } from 'react-router-dom';
 import { sortable } from '@patternfly/react-table';
 import * as classNames from 'classnames';
 import { getMachineAWSPlacement, getMachineRole, getMachineSetInstanceType } from '@console/shared';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { RowProps, TableColumn } from '@console/dynamic-plugin-sdk';
 import { Tooltip, Button } from '@patternfly/react-core';
 import { PencilAltIcon } from '@patternfly/react-icons';
 import { useTranslation } from 'react-i18next';
 
-import { MachineAutoscalerModel, MachineModel, MachineSetModel } from '../models';
-import { K8sKind, MachineDeploymentKind, MachineSetKind, referenceForModel } from '../module/k8s';
+import { MachineAutoscalerModel, MachineModel, MachineSetModel, NodeModel } from '../models';
+import {
+  K8sKind,
+  MachineDeploymentKind,
+  MachineSetKind,
+  MachineKind,
+  NodeKind,
+  referenceForModel,
+  Selector as SelectorType,
+} from '../module/k8s';
 import { MachinePage } from './machine';
 import { configureMachineAutoscalerModal, configureReplicaCountModal } from './modals';
-import { DetailsPage, ListPage, Table, TableData, RowFunctionArgs } from './factory';
+import { DetailsPage, TableData } from './factory';
+import VirtualizedTable from './factory/Table/VirtualizedTable';
```

Comment:
@rebeccaalpert I'm not sure if this is in the scope of this pr...maybe @spadgett would know. Are we supposed to be referencing api's from the shared package for the purpose of providing core only vs admin plugins?